### PR TITLE
fix a few coverity warnings

### PR DESCRIPTION
**Pull request by mkortstiege**

### CODE REVIEW

**Comment by mkortstiege** [comment]
jenkins build this please

**Comment by mkortstiege** [comment]
@Paxxi mind having a look?

File: xbmc/utils/StreamDetails.cpp [hunk 1]
```diff
@@ -294,6 +294,9 @@ int CStreamDetails::GetSubtitleStreamCount(void) const
 
 CStreamDetails::CStreamDetails(const CStreamDetails &that)
 {
+  m_pBestVideo = NULL;
+  m_pBestAudio = NULL;
+  m_pBestSubtitle = NULL;
```
Comment (Paxxi) [inline xbmc/utils/StreamDetails.cpp]: prefer nullptr over NULL for new code

File: xbmc/windows/GUIMediaWindow.cpp [hunk 2]
```diff
@@ -1593,8 +1593,7 @@ bool CGUIMediaWindow::OnContextButton(int itemNumber, CONTEXT_BUTTON button)
   case CONTEXT_BUTTON_BROWSE_INTO:
     {
       CFileItemPtr item = m_vecItems->Get(itemNumber);
-      if(Update(item->GetPath()))
-        return true;
+      Update(item->GetPath());
```
Comment (Paxxi) [inline xbmc/windows/GUIMediaWindow.cpp]: I'm wondering about this one, do we only return false for unhandled messages or should it return false if Update fails?

Comment (mkortstiege) [inline xbmc/windows/GUIMediaWindow.cpp]: No idea really without looking at the code. But this is not changing logic.

Comment (Paxxi) [inline xbmc/windows/GUIMediaWindow.cpp]: No it's good to go as is, just curious if this was originally a bug or just weird :)

Comment (mkortstiege) [inline xbmc/windows/GUIMediaWindow.cpp]: Just weird.. that weird that even coverity pointed it out ;) Will update to nullptr later today or tomorrow.

**Comment by mkortstiege** [comment]
Updated according comments and rebased.

**Comment by mkortstiege** [comment]
jenkins build this please

**Comment by mkortstiege** [comment]
jenkins build this please

**Pull request merged**